### ISSUE TITLE
add ARIA-live to progress indicator

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#1028]
### Description

We have been recommended to add a [ARIA-live](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) attribute to our progress indicator. This will tell screen readers to update the users when it changes. In our case this will be on page change. The attribute should be set to `polite`
![Skjermbilde 2023-03-22 kl  14 51 00](https://user-images.githubusercontent.com/33222679/226925215-4e95cd12-ff1c-4249-b3b2-7494a4f0612d.png)

## Specification:
- [ ] Are there any negative outcomes if we include this?

**Comment by StianVestli** [comment]
org/ssb
